compiler/crates/common/src/feature_flags.rs

```diff
@@ -124,8 +124,44 @@ pub struct FeatureFlags {
     /// Disable validation of the `edgeTypeName` argument on `@prependNode` and `@appendNode`.
     #[serde(default)]
     pub disable_edge_type_name_validation_on_declerative_connection_directives: FeatureFlag,
+
+    /// Adds @ts-nocheck to generated Typescript Files
+    #[serde(default = "default_true")]
+    pub typescript_disable_checking_generated_files: bool,
+}
+
+impl Default for FeatureFlags {
+    fn default() -> Self {
+        Self {
+            enable_relay_resolver_transform: false,
+            enable_catch_directive_transform: Default::default(),
+            relay_resolver_enable_interface_output_type: Default::default(),
+            no_inline: Default::default(),
+            enable_3d_branch_arg_generation: false,
+            actor_change_support: Default::default(),
+            text_artifacts: Default::default(),
+            skip_printing_nulls: Default::default(),
+            enable_fragment_aliases: Default::default(),
+            enforce_fragment_alias_where_ambiguous: Default::default(),
+            compact_query_text: Default::default(),
+            enable_resolver_normalization_ast: false,
+            enable_relay_resolver_mutations: false,
+            enable_strict_custom_scalars: false,
+            allow_resolvers_in_mutation_response: Default::default(),
+            allow_required_in_mutation_response: Default::default(),
+            disable_resolver_reader_ast: false,
+            enable_fragment_argument_transform: false,
+            allow_resolver_non_nullable_return_type: Default::default(),
+            disable_schema_validation: false,
+            disallow_required_on_non_null_fields: false,
+            prefer_fetchable_in_refetch_queries: false,
+            disable_edge_type_name_validation_on_declerative_connection_directives: Default::default(),
+            typescript_disable_checking_generated_files: default_true(),
+        }
+    }
```

Comment:
I think it's okay if we skip this for now and rely on the derived default. I think it's okay to let all places that use default feature flags from `Default::default()` in Rust instead of via serde (just tests I think?) use `false` as the default here.

I assume that's what you started with. Was there some reason it was problematic to let those tests default to false?